nikola/data/themes/base/templates/comments_helper_discourse.tmpl

```diff
@@ -0,0 +1,31 @@
+## -*- coding: utf-8 -*-
+
+<%! import json %>
+
+<%def name="comment_form(url, title, identifier)">
+    %if comment_system_id:
+        <div id="discourse-comments"></div>
+        <script type="text/javascript">
+            DiscourseEmbed = { discourseUrl: '${blog_url}',
```

Comment:
This should probably be the Discourse forum’s URL, not the blog URL, please double-check. The Discourse URL can be placed in the comment_system_id setting (as done by e.g. isso), this should also be documented in manual.rst.

Reply:
you are right I have to test it asap